apps/demo-ui/ui/src/App.tsx

```diff
@@ -75,7 +75,7 @@ export default function App() {
         previousInteractions = [...previousInteractions, chat]
       })
       console.log("Setting previous interactions", previousInteractions)
-      setChatHistory(previousInteractions)
+      setChatHistory(previousInteractions.reverse())
```

Comment:
Can you just update the order in line 75?

Reply:
Addressed in PR #394